### ISSUE TITLE
Hyperparameter Tuning

### ISSUE THREAD

**Issue by jachang0628** [jachang0628/cat_pig_guess#1]
* Try out different layers/neurons/activation function combinations
* Tune the learning rate as well as other hyperparameters via Ax
